### PR TITLE
qr_code_frame_calc branch: code review + tests

### PR DESCRIPTION
**Pull request by tadeubas**
Doing code review and tests on the new branch `qr_code_frame_calc`

### CODE REVIEW

File: src/krux/pages/addresses.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def list_address_type(self, addr_type=0):
                     self.ctx.display.clear()
                     self.ctx.display.draw_centered_text(loading_txt % (num_checked + 1))
 
-                    pos_str = str(num_checked + 1) + ". "
+                    pos_str = str(num_checked + 1) + "." + " "  # thin space
```
Comment (jdlcdl) [inline src/krux/pages/addresses.py:94]: I'm not seeing the difference in these two values.  Is it just to be more explicit that the space is needed, so it is not carelessly removed in the future?

Comment (odudex) [reply src/krux/pages/addresses.py:94]: Thin space is used as a trick. Krux line breaker will split words detecting spaces, but "thin spaces" won't be detected. On this case Tadeu found out multisig address wouldn't fit on Dock's screen, thin space will ensure the address number prefix will be in the same line as the address start (not in a exclusive line) so the screen space will be better used and address will fit.

Comment (tadeubas) [reply src/krux/pages/addresses.py:94]: @jdlcdl here you can see the unicode character: https://www.compart.com/en/unicode/U+2009

Comment (jdlcdl) [reply src/krux/pages/addresses.py:94]: Nice! I didn't even realize it was not a normal space character.  Thank you for the explanation.

File: src/krux/qr.py [hunk 2]
```diff
@@ -88,7 +95,7 @@ def total_count(self):
             # Single-part URs have no expected part indexes
             if self.decoder.fountain_decoder.expected_part_indexes is None:
                 return 1
-            return self.decoder.expected_part_count()
+            return self.decoder.expected_part_count() * 2
```
Comment (odudex) [inline src/krux/qr.py:98]: Where `*2` comes from? 
`fountain_decoder.py` has an `estimated_percent_complete()` which uses:
`estimated_input_parts = self.expected_part_count() * 1.75`

Comment (tadeubas) [reply src/krux/qr.py:98]: The 1.75 is an estimative they use, but it is wrong! In the end the scan will only finish after reading X estimated parts + X index parts... so it is 2 times the estimated parts

**Pull request merged**